testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/microprofile/opentracing/ResourceWithCustomOperationNameBeanTestCase.java

```diff
@@ -54,7 +55,13 @@
     }
 
     @Test
+    @Ignore
```

Comment:
General comment: @Ignore("some reasoning here ...") is helpful as TeamCity can show these messages and we have testsuite analyzer which looks into this when we are looking into details about ignored or assumed tests